src/list.json

```diff
@@ -120,7 +120,16 @@
     ],
     "N": [],
     "O": [],
-    "P": [],
+    "P": [
+      {
+        "organization": "Pusher",
```

Comment:
hey mate!
Please Re-style it via this command 
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2332883/patch | git am`
`git push `

Reply:
I am creating a new pull request with the changes.

Reply:
It is still failing . The message in terminal says "fatal: previous rebase directory .git/rebase-apply still exists but mbox given."

Reply:
@front-runner-sd because as the message say the .git/rebase-apply already exist in that directory so you need to remove the repo directory and git clone it again then enter the command above.
Sorry to say but Pusher has already been added by some one so I close this PR